File: api/internal/models/university.go
package models

type University struct {
	ID                int    `json:"id" db:"id"`
	Name              string `json:"name" db:"name"`
	FullName          string `json:"full_name" db:"full_name"`
	DirectionsPageURL string `json:"directions_page_url" db:"directions_page_url"`
}

File: directions_parser/models/university.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship

from models.base import Base


class University(Base):
    __tablename__ = "universities"

    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)
    directions_page_url = Column(String, nullable=False)
    children = relationship('Direction')
